EIPS/eip-draft_dtype_alias.md

```diff
@@ -0,0 +1,86 @@
+---
+eip: <to be assigned>
+title: dType Alias Extension - Decentralized Type System
+author: Loredana Cirstea (@loredanacirstea), Christian Tzurcanu (@ctzurcanu)
+discussions-to: https://github.com/ethereum/EIPs/issues/2192
+status: Draft
+type: Standards Track
+category: ERC
+created: 2019-07-16
+requires: 1900, 2157
+---
+
+## Simple Summary
+
+The dType Alias is a system for providing human-readable resource identifiers to on-chain content, supporting any type of data.
+
+## Abstract
+
+The dType Alias is a system for providing human-readable resource identifiers to on-chain content. A resource identifier is based on the type of data (identifier provided by dType, EIP-1900) and the data content (identifier provided by dType Storage Contracts, EIP-2157). It is a universal way of addressing content, supporting any type of data.
```

Comment:
I think the `EIP-1900` and `EIP-2157` part should be proper links to eips.ethereum.org